integration-tests/ledger-signer-failing-tests.spec.ts

```diff
@@ -37,7 +37,7 @@ CONFIGS().forEach(({ lib, setup }) => {
                 done();
             });
 
-            it('Should throw error when user declines to sign', async (done) => {
+            it('As a User I want to verify that Ledger will throw an error when I declined to sign', async (done) => {
```

Comment:
Should we use the same format as the other files "Verify that..." without "As a User I want to"

Reply:
changed to
'Verify that Ledger will throw an error when when signing is declined'